app/models/Computer.java

```diff
@@ -60,23 +60,15 @@
 
     public List<ValidationError> validate() {
 
-        List<ValidationError> errors = new ArrayList<ValidationError>();
+        // Note that changes in this model where NOT required, I made them to keep code cleaner.
+        // I.e. You don't need to create getters for fields like name, or id as they are public
 
-        //if(Computer.find.where().eq("name", getName()).findRowCount() != 0){
-        Logger.debug("current id: " + getId());
-        if(Computer.find.where().eq("name", getName()).ne("id", getId()).findRowCount() != 0){
+        List<ValidationError> errors = new ArrayList<>();
+        if (Computer.find.where().ilike("name", name).ne("id", id).findRowCount() != 0) {
```

Comment:
I think another problem is that when this validate function gets called on `create()`, the `id` value is still null